### PR TITLE
feature/layout

### LINKED ISSUE #5: Create a 2 column layout with dummy events and calendar

### PR DESCRIPTION
**Pull request by rednur01**
- feat(layout): dummy text 2 layout view
- feat(ui): Create ui structure
- chore(code): Leave todo comment for code cleanup

### CODE REVIEW

File: src/ui/calendar-panel.zig [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+        const window = self.vxWin.child(.{
+            .x_off = (self.vxWin.width / 2) + 1,
+            .y_off = 0,
+            .width = .{ .limit = self.vxWin.width / 2 },
+            .border = .{
+                .where = .all,
+                .style = .{ .fg = calendarPanelBorderColor },
+            },
+        });
```
Comment (Uzaaft) [inline src/ui/calendar-panel.zig:37]: The ChildOptions struct thats the input of the vxWin.child function has fields that are usize. This makes it so the potential floats are being cast to usize. We should floor them but for now I didnt do that. 

```suggestion
        const x_off: usize = @intFromFloat((self.vxWin.width / 2) + 1);
        const y_off: usize = 0;
        const width_limit: usize = @intFromFloat(self.vxWin.width / 2);


        const window = self.vxWin.child(.{
            .x_off = x_off,
            .y_off = y_off,
            .width = .{ .limit = width_limit },
            .border = .{
                .where = .all,
                .style = .{ .fg = calendarPanelBorderColor },
            },
        });
```

File: src/app.zig [hunk 2]
```diff
@@ -76,9 +82,18 @@ pub const App = struct {
...
                 if (key.matches('c', .{ .ctrl = true }))
                     self.should_quit = true;
+                if (key.matches('l', .{ .ctrl = true }))
+                    if (self.state.activePanel != Panel.calendar) {
+                        self.state.activePanel = Panel.calendar;
+                    };
+                if (key.matches('h', .{ .ctrl = true }))
+                    if (self.state.activePanel != Panel.events) {
+                        self.state.activePanel = Panel.events;
+                    };
```
Comment (Uzaaft) [inline src/app.zig:96]: We should probably for ease of use also allow arrows.

File: src/ui/events-panel.zig [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+        const window = self.vxWin.child(.{
+            .x_off = 0,
+            .y_off = 0,
+            .width = .{ .limit = self.vxWin.width / 2 },
+            .border = .{
+                .where = .all,
+                .style = .{ .fg = eventsPanelBorderColor },
+            },
+        });
```
Comment (Uzaaft) [inline src/ui/events-panel.zig:37]:
```suggestion
	const x_off: usize = 0;
        const y_off: usize = 0;
        const width_limit: usize = @intFromFloat(self.vxWin.width / 2);



        const window = self.vxWin.child(.{
            .x_off = x_off,
            .y_off = y_off,
            .width = .{ .limit = width },
            .border = .{
                .where = .all,
                .style = .{ .fg = eventsPanelBorderColor },
            },
        });
```

Comment (Uzaaft) [reply src/ui/events-panel.zig:37]: Same as above

File: src/app.zig (hunk 2 above)
Comment (rednur01) [reply src/app.zig:96]: I'm going to separate the keybinds module so I'll add the arrow keys then. Also this way of doing it won't work, I'll separate the "actions" and then bind each key to an action.

**Pull request merged**